sites/public/pages/listings.tsx

```diff
@@ -114,6 +117,30 @@ const ListingsPage = () => {
               options={preferredUnitOptions}
               defaultValue={filterState?.bedrooms?.toString()}
             />
+            <Field
+              id="zipCodeField"
+              name={ListingFilterKeys.zipcode}
+              label={t("listingFilters.zipCode")}
+              register={register}
+              controlClassName="control"
+              placeholder={t("listingFilters.zipCodeDescription")}
+              validation={{
+                minLength: 5,
+                pattern: /[0-9 ,]*/,
+              }}
+              error={errors.zipCodeField}
+              errorMessage={t("errors.multipleZipCodeError")}
```

Comment:
I'm not able to trigger this error message being shown. When I input something that is less than 5 characters, clicking "Apply Filters" doesn't do anything (modal is still there, no error message). And for other invalid inputs that are 5 characters or more (like "hello"), when I click "Apply Filters" the modal goes away and it seems to have just "applied" the filters b/c there are no listings there.

Did you find a way to have this error message show up?

Reply:
No, this is the standard way of doing this, but I couldn't get the error to actually show up, despite many different variations. I opened #429 to fix this and think maybe I should get help from someone who's more experienced in React and just check in this filter as-is.